test/plugins/features/movies/controller.test.js

```diff
@@ -2,18 +2,56 @@
 
 const Controller = require('../../../../lib/plugins/features/movies/controller');
 
-describe('movie controller', () => {
+describe('POST /movie controller', () => {
```

Comment:
Typically, we keep a single top level describe block with just "movie controller", and then nest the different functions we are testing within that single describe block. For example:

```js
describe('movie controller', () => {

  describe('create', () => {
    ...
  });

  describe('list', () => {
    ...
  });

});
```